test/specs/sortByAlphabetTest.regression.js

```diff
@@ -0,0 +1,22 @@
+import LoginPage from '../pageobjects/login.page.js'
+import HomePage from '../pageobjects/home.page.js'
+
+const executeSortTest = async (sortOption, expectedValue) => {
+  await LoginPage.open()
+  await LoginPage.login(process.env.USERNAME1, process.env.PASSWORD)
+
+  await HomePage.sortByAlphabet(sortOption)
+  await expect(HomePage.productSortContainer).toHaveValue(expectedValue)
```

Comment:
Let's also do additional verification for checking actual item order, since there could theoretically be a bug where the sorter says it's completed the sort but nothing changes in the actual order.

Reply:
Added verification for sorting by price